package.json

```diff
@@ -78,7 +78,11 @@
 		"test-dangerci-and-adminpage-and-extensions": "yarn danger ci && yarn test-adminpage && yarn test-extensions",
 		"test-search": "jest modules/search",
 		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json}\"",
-		"version-packages": "bash ./bin/version-packages.sh"
+		"version-packages": "bash ./bin/version-packages.sh",
+		"tunnel": "[ -f ./docker/bin/jt/tunnel.sh ] && ./docker/bin/jt/tunnel.sh",
+		"tunnel:break": "[ -f ./docker/bin/jt/tunnel.sh ] && ./docker/bin/jt/tunnel.sh break",
+		"tunnel:config": "[ -f ./docker/bin/jt/config.sh ] && ./docker/bin/jt/config.sh",
+		"tunnel:install": "echo 'Auto-install is not available at the moment. Please visit https://github.com/Automattic/jurassic-tube for installation instructions.'"
```

Comment:
I note that `yarn tunnel` just says "error Command failed with exit code 1" if JT hasn't been installed. Is it worth trying to do better, pointing to `yarn tunnel:install` or something?

Maybe
```
"tunnel": "[ -f ./docker/bin/jt/tunnel.sh ] || { echo 'Use `yarn tunnel:install` to enable tunneling.'; exit; }; ./docker/bin/jt/tunnel.sh",
```
and so on.

P.S. I checked, that syntax is valid for [POSIX shell compound commands](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_09_04).

Reply:
Thanks @anomiex, that's a huge improvement 👍